main.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+###
+# EXIT CODES
+# 0 - success
+# 1 - parameter error
+# 100 - new tag found, pr_script triggered
+###
+
+# no REQUIRED_ENV_VARS
+
+# clean past log
+rm -f $DATA_DIR/last-update-result.log
+
+# vars
+CONFIG_FILEPATH='config.json'
+
+# seed .env if we are not in test mode
+if [[ -z "${TEST_ENV_CONFIG}" ]]; then
+    source .env
+else
+    echo "[INFO] TEST CONFIG SEEDED"
```

Comment:
This is a bit misleading, the `test.env` is sourced before launching this script so the script just does not know.  Maybe just "[INFO] Not reading production config".

Reply:
I've changed it for custom env and config file passed by parameters, instead. 
It was a misleading way to handle env file, I agree.